.github/workflows/pull_request_check.yaml

```diff
@@ -0,0 +1,28 @@
+name: Pull Request Check
+on:
+  pull_request:
+    types: [opened, edited, reopened, synchronize]
+jobs:
+  pull_request_check:
+    name: pull request check
+    runs-on: ubuntu-latest
+    steps:
+      - uses: nearform/github-action-check-linked-issues@v1
+        id: check-linked-issues
+        with:
+          exclude-branches: "release_v/**, backport_v/**, main, latest-dep-update-/**, min-dep-update-/**, dependabot/**"
+          comment: false
+          github-token: ${{ secrets.REPO_SCOPED_TOKEN }}
+      - name: Print helpful error message
+        if: ${{ steps.check-linked-issues.outputs.linked_issues_count }} == "0" }}
+        run: |
+          msg="Linked issues: ${{ steps.check-linked-issues.outputs.linked_issues }}"
```

Comment:
Will this do anything since we only run this if the linked issue count is 0? If I'm following correctly...

Reply:
If no issue is linked on a pull request, it will print a helpful error message in the CI workflow output